config/credentials.yml.default

```diff
@@ -0,0 +1,17 @@
+mysql:
```

Comment:
There are no environments set. 
In database.yml they are referenced with environments:
<%= Rails.application.credentials.dig(:mysql, :staging, :password ) %>